static/styles/lib/base.scss

```diff
@@ -450,3 +450,7 @@ form.inline {
 	background-color: transparent;
 	border: none;
 }
+
+select {
```

Comment:
really set all select width to 100%?

Reply:
100% width of the parent container. :-) so they are not too narrow when there are no options provided.